src/index.html

```diff
@@ -18,5 +30,91 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <div>
+        <a
+          class="logo-header"
+          href="#"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            data-qa="hover"
+            class="nav__item"
+          >
+            <a
+              data-qa="hover"
```

Comment:
The `data-qa` attribute is used twice on the same `<li>` and `<a>` elements. It's redundant to have the same attribute on both the parent and child elements unless they serve different purposes. Consider removing one of them if they are not needed for different functionalities.